test/test_smartfile.py

```diff
@@ -0,0 +1,57 @@
+import os
+import unittest
+from StringIO import StringIO
+
+from smartfile import BasicClient
+from smartfile.errors import ResponseError
+
+API_KEY = os.environ.get("API_KEY")
+API_PASSWORD = os.environ.get("API_PASSWORD")
+TESTFN = "testfn"
```

Comment:
Might want to move this variable under the logic for checking if API_KEY and API_PASSWORD are populated. It'll help with readability.